ecosystem/sep-0008.md

```diff
@@ -225,17 +225,56 @@ Name | Type | Description
 `status` | string | `"action_required"`
 `message` | string | A human readable string containing information regarding the action required.
 `action_url` | string | A URL that allows the user to complete the actions required to have the transaction approved.
+`action_method` | string | (Optional) `GET` or `POST`, indicating the type of request that should be made to the `action_url`. If not provided, `GET` is assumed.
+`action_fields` | string[] | (Optional) An array of additional fields defined by [SEP-9 Standard KYC / AML fields] that the client may optionally provide to the approval service when sending the request to the `action_url`.
 
 ###### Example
 
 ```json
 {
   "status": "action_required",
   "message": "Please sign the terms of service via the provided URL.",
-  "action_url": "https://..."
+  "action_url": "https://...",
+  "action_method": "POST",
+  "action_fields": ["email_address", "mobile_number"]
 }
 ```
 
+###### Action URL
+If the `action_method` is provided the client uses the type of request indicated when sending a request to the `action_url`. If `action_fields` are to be provided and the fields contain sensitive or personal information the `POST` case is preferred to ensure sensitive information is not transmitted in the URL.
```

Comment:
Adding a comma helps me understand it better.

```suggestion
If the `action_method` is provided, the client uses the type of request indicated when sending a request to the `action_url`. If `action_fields` are to be provided and the fields contain sensitive or personal information, the `POST` case is preferred to ensure sensitive information is not transmitted in the URL.
```

Reply:
https://github.com/DSTOQ/stellar-protocol/pull/4